src/test/java/lotto/domain/LottoCheckerTest.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoCheckerTest {
+
+    @Test
+    @DisplayName("내가 산 복권과 당첨 복권이 3개 일치")
+    void countMatch() {
+        Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
        Lotto lotto = new Lotto(1, 2, 3, 4, 5, 6);
```
Lotto를 생성하는 개발자 입장에서 위와 같이 구현할 수 있도록 지원하면 편리하지 않을까?
항상 객체와 메서드를 구현할 때 이 객체와 메서드를 사용하는 개발자 입장을 고려하면서 구현해 보는 습관을 들일 것을 추천해본다.

힌트: 가변인자